test/clt-tests/core/test-skipping-plugin-loading.rec

```diff
@@ -0,0 +1,194 @@
+––– output –––
+--------------
+SHOW TABLES
+--------------
+--------------
+DROP TABLE IF EXISTS tbl
+--------------
+--------------
+SHOW TABLES
+--------------
+--------------
+CREATE TABLE tbl (id BIGINT, model TEXT, storage_capacity INTEGER, color string, release_year INTEGER, price FLOAT, discounted_price FLOAT, sold BOOL, date_added TIMESTAMP, product_codes MULTI, values MULTI64, additional_info JSON, vector float_vector knn_type='hnsw' knn_dims='4' hnsw_similarity='l2')
+--------------
+––– input –––
+chmod +x ./test/clt-tests/core/generate-1mln-records.sh
+––– output –––
+––– input –––
+total_iterations=20 ./test/clt-tests/core/generate-1mln-records.sh
+––– output –––
++-------+------+
+| Index | Type |
++-------+------+
+| tbl   | rt   |
++-------+------+
++----------+
+| count(*) |
++----------+
+|     1000 |
++----------+
+––– input –––
+mysql -h0 -P9306 -e "REPLACE INTO tbl SET model ='iPhone 15 PRO' WHERE id = 101; select * from tbl WHERE id = 101;"
+––– output –––
++------+---------------+------------------+--------+--------------+-------------+------------------+------+------------+---------------+----------------------------------------+-----------------------------------------------------------------------------------+-------------------------------------+
+| id   | model         | storage_capacity | color  | release_year | price       | discounted_price | sold | date_added | product_codes | values                                 | additional_info                                                                   | vector                              |
++------+---------------+------------------+--------+--------------+-------------+------------------+------+------------+---------------+----------------------------------------+-----------------------------------------------------------------------------------+-------------------------------------+
+|  101 | iPhone 15 PRO |              256 | silver |         2021 | 1099.989990 |       989.989990 |    1 | 2224442480 | 1,2,3         | 523456764345678976,9223372036854775807 | {"features":["ProMotion display","A15 Bionic chip","Ceramic Shield front cover"]} | 0.773448,0.312478,0.137971,0.459821 |
++------+---------------+------------------+--------+--------------+-------------+------------------+------+------------+---------------+----------------------------------------+-----------------------------------------------------------------------------------+-------------------------------------+
+––– input –––
+searchd --stopwait
+––– output –––
+Manticore %{SEMVER} %{COMMITDATE}#!/(\sdev)?\s/!#(columnar %{SEMVER} %{COMMITDATE}) (secondary %{SEMVER} %{COMMITDATE}) (knn %{SEMVER} %{COMMITDATE})
```

Comment:
Do we have any reusable blocks for Manticore? When something changes in the output of starting it, we'll need to replace it in almost all files. It's better to use reusable blocks here.

Reply:
Created a new block, added it to the test